src/algebra/group_action_hom.lean

```diff
@@ -0,0 +1,252 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+import algebra.group_ring_action
+
+/-!
+# Equivariant homomorphisms
+
+## Main definitions
+
+* `mul_action_hom M X Y`, the type of equivariant functions from `X` to `Y`, notated `X →[M] Y`.
```

Comment:
Please put the notation in a different list under `## Notation`. I think this will help with (future) doc parsing/generating tools.

Reply:
done